tcms/testplans/models.py

```diff
@@ -94,15 +94,22 @@ def list(cls, query=None):
     def confirmed_case(self):
         return self.case.filter(case_status__name='CONFIRMED')
 
-    def add_case(self, case, sortkey=0):
+    def add_case(self, case, sortkey=None):
 
-        tcp, is_created = TestCasePlan.objects.get_or_create(
+        if sortkey is None:
+            lastcase = self.testcaseplan_set.order_by('-sortkey').first()
+            if lastcase and lastcase.sortkey is not None:
+                sortkey = lastcase.sortkey + 10
+            else:
+                sortkey = 0
```

Comment:
This can be moved as a stand alone if statement:

```
if sortkey is None:
    sortkey=0
````

and contribute to simplifying the expression below.